doc/modules/language-guide/pages/about-this-guide.adoc

```diff
@@ -77,7 +77,7 @@ As a counterpoint to the core values and goals, the engineering organization als
 
 . Having a more advanced type system, with more complex types.
 . Simplicity over functionality in design or implementation (the "Worse is Better" approach).
-. Interoperability with existing smart contract platforms.
+. Running on platforms besides the Internet Computer
```

Comment:
```suggestion
. Support for other platforms than the Internet Computer
```

Reply:
Too late :-)

(and I intentionally wrote it this way. Once the IC has “support for ethereum”, whatever that means, we’d have that too.)